src/styles.css

```diff
@@ -62,6 +102,26 @@ body {
   max-height: 60vh;
 }
 
+#ul-container-list li {
+  list-style: none;
+  display: flex;
+  gap: 10px;
+  padding: var(--padding-table-content);
+  border-bottom: var(--border-bottom-list);
+}
+
+#ul-container-list li i {
+  margin-left: auto;
+}
+
+#clearbtn-list {
+  border: none;
+  width: var(--width-table-content);
+  padding: var(--padding-table-content);
+  color: rgba(128, 128, 128, 0.622);
+  cursor: pointer;
+}
+
 .delet-Container-eachelement {
   background-color: rgba(32, 29, 126, 0.084);
   backdrop-filter: blur(0.2rem);
```

Comment:
- [ ] `required`
- [ ] Amazing job so and going above and beyond on your project 👏 👏
- [ ] However, you missed one of the requirements.  the requirements states as follows
- Adding a new item.
- Removing a selected item.
- Marking a selected item as complete.
- Removing all items marked as complete at once.
- Reordering a selected item (as drag-and-drop) [as an additional feature].

- [ ] kindly make amends on the pop-up and just stick to the requirement given

![image](https://user-images.githubusercontent.com/66230306/184508416-4c3cd708-0228-43e5-9ea1-018b6a34098b.png)